docs/en/client/java-sdk-user-guide.md

```diff
@@ -1372,4 +1372,4 @@ The interface is `com.ctrip.framework.apollo.spi.ConfigServiceLoadBalancerClient
 
 The Input is multiple ConfigServices returned by meta server, and the output is a ConfigService selected.
 
-The default service provider is `com.ctrip.framework.apollo.spi.RandomConfigServiceLoadBalancerClient`, which chooses one ConfigService from multiple ConfigServices using random strategy .
+The default service provider is `com.ctrip.framework.apollo.spi.RandomConfigServiceLoadBalancerClient`, which chooses one ConfigService from multiple ConfigServices using random strategy .
```

Comment:
I see many changes in the `zh/client/java-sdk-user-guide.md`. Shouldn't we apply these to the `en/client/java-sdk-user-guide.md` as well?

Reply:
I will synchronize the English part when the Chinese part is finalized